browser/brave_features_internal_names.h

```diff
@@ -6,7 +6,14 @@
 #ifndef BRAVE_BROWSER_BRAVE_FEATURES_INTERNAL_NAMES_H_
 #define BRAVE_BROWSER_BRAVE_FEATURES_INTERNAL_NAMES_H_
 
-constexpr char kBraveVPNFeatureInternalName[] = "brave-vpn";
+#include "brave/components/brave_vpn/buildflags/buildflags.h"
```

Comment:
missing build_config.h for `IS_WIN`

Reply:
done in https://github.com/brave/brave-core/pull/15989/commits